### PR TITLE
Add logs to help triage Firestore task update failure

### PR DESCRIPTION
**Pull request by keyonghan**
Help with https://github.com/flutter/flutter/issues/143497

### CODE REVIEW

File: app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart [hunk 1]
```diff
@@ -144,12 +144,16 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
   }
 
   /// Queries the task document and updates based on the latest build data.
-  Future<void> updateFirestore(Build build, String sha, String taskName) async {
+  Future<void> updateFirestore(Build build, String commitKeyId, String taskName) async {
```
Comment (yusuf-goog) [inline app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart:147]: Should we name the arg consistently (rawCommitKey)?

**Review by yusuf-goog — APPROVED** [review]
LGTM with one question

File: app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart (hunk 1 above)
Comment (keyonghan) [reply app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart:147]: From the user_data, it uses `rawCommitKey`, but this is actually the Key.id of a Commit.

File: app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart [hunk 2]
```diff
@@ -144,12 +144,16 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
...
     final FirestoreService firestoreService = await config.createFirestoreService();
+    final String sha = commitKeyId.split('/').last;
     final String documentName = '$kDatabase/documents/tasks/${sha}_${taskName}_1';
+    log.info('getting firestore document: $documentName');
     final f.Task firestoreTask =
         await f.Task.fromFirestore(firestoreService: firestoreService, documentName: documentName);
+    log.info('updating firestoreTask based on build');
     firestoreTask.updateFromBuild(build);
+    log.info('finished updating firestoreTask based on builds');
```
Comment (ricardoamador) [inline app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart:156]: For these two log messages can we add some kind of identifier? Something like the commit for easier debugging?

Comment (keyonghan) [reply app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart:156]: The `commit` info is included in line 151. These two logs are just used as a way to tell which step in between fails if any. In other words, if we see line 154 or line 156, line 151 should already show up right above.

**Pull request merged**